FB.py

```diff
@@ -0,0 +1,82 @@
+import unittest
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import Select
+from selenium.common.exceptions import TimeoutException
+import time
+class SignUpTest(unittest.TestCase):
+    def setUp(self):
+        self.driver= webdriver.Firefox(executable_path="/home/ruttaba/PycharmProjects/geckodriver")
+        self.driver.get("https://www.facebook.com/")
+        self.assertIn("Facebook", self.driver.title)
+        self.driver.maximize_window()
+    def test_enter_credentials_in_signUp_form(self):
+        #review test fb git
+        driver = self.driver
+        firstName = 'Ruttaba'
+        surname = 'Assad'
+        emailId = 'Ruttaba@gmail.com'
+        newPassword = 'qwerty1234'
+        #CSS Selector
+        enter_name = 'firstname'
+        enter_surname = 'lastname'
+        enter_email_Id = 'u_0_r'
+        enter_newPass_Id = 'u_0_w'
+        reEnter_email = 'u_0_u'
```

Comment:
@ruttaba Please consult python naming guides. "reEnter" is not according to python conventions. Also rename other attributes which are not according to PEP8 standards.